calour/heatmap/heatmap.py

```diff
@@ -86,10 +86,55 @@ def _truncate_middle(x, length=16):
         return ['%s..%s' % (i[:mid], i[-mid:]) if len(i) > length else i for i in y]
 
 
+def _set_axis_ticks(ax, which, ticklabels, tickmax, n, kwargs, ticklabel_len):
```

Comment:
why is kwargs not **kwargs? and why is it before ticklabel_len?
(probably my lack of knowledge in python...)

Reply:
it is you lack of python knowledge :). If you don't need to expand the dict into key word arguments, you shouldn't use `**`